design-patterns/creational-patterns/src/builder/StoneHouseBuilder.java

```diff
@@ -0,0 +1,58 @@
+package builder;
+
+public class StoneHouseBuilder implements HouseBuilder {
+	private House result;
```

Comment:
the name of the reference should be 'house' or something similar

Reply:
Renamed.